assets/js/pages/AnalyticsEula/AnalyticsEulaModal.stories.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+
+import Button from '@common/Button/Button';
+import AnalyticsEulaModal from './AnalyticsEulaModal';
+
+export default {
+  title: 'Patterns/AnalyticsEulaModal',
+  component: AnalyticsEulaModal,
+  argTypes: {
+    checked: {
+      type: 'boolean',
+      description: 'Checks checkbox',
+    },
+    isOpen: {
+      type: 'boolean',
+      description: 'Sets the visibility of the modal',
+    },
+  },
+};
+
+function ButtonToOpenModal({ ...rest }) {
```

Comment:
question:
do you think we need all this to show the modal itself?
We could simply have a static modal view, we don't need to test that it shows up when a button is clicked.
Take the `ActivityLogDetailModal` as reference for example.

We could simply have:
```
export const Default = {
  args: {
    open: false,
  },
};
```

Reply:
There is a combination of solutions in our current Storybook for modal behaviour (`ApiKeySettingsModal` VS `ActivityLogDetailModal`). I am happy to change this one to function like you are suggesting.